db_facts/__init__.py

```diff
@@ -1,2 +1,10 @@
+__all__ = [
+    'db',
+    'DBFacts',
+    'UserErrorException'
+]
```

Comment:
Using `__all__` helps clarify the public API and also gives Sphinx the hint to put the documentation for these in the overall package.  These are pretty fundamental, so I'm more comfortable having them in the top-level package than I am with the names of the subpackages they currently live in :)